iCount/externals/star.py

```diff
@@ -200,6 +200,8 @@ def map_reads(reads, genome_index, out_dir, annotation='', multimax=10, mismatch
         '--outFilterMultimapNmax', '{:d}'.format(multimax),
         '--outFilterMismatchNmax', '{:d}'.format(mismatches),
         '--alignEndsType', 'EndToEnd',
+        ## Share memory
+        '--genomeLoad', 'LoadAndRemove',
         # otherwise soft-clipping of the starts and ends may produce too
```

Comment:
This comment belongs to `--alignEndsType` line, so put you additional two lines after it.